website/scaleway.erb

```diff
@@ -73,6 +73,16 @@
           </ul>
         </li>
 
+        <li<%= sidebar_current("docs-scaleway-resource-storage") %>>
```

Comment:
check with @QuentinBrosse but i think `sidebar_current` is deprecated

Reply:
Yeps, that's correct. I must update this PR since we have merged Quentin's.